californium-core/src/main/java/org/eclipse/californium/core/network/interceptors/HealthStatisticLogger.java

```diff
@@ -183,7 +183,7 @@ public void dump() {
 	}
 
 	@Override
-	public void sendRequest(Request request) {
+	public void requestSent(Request request) {
 		if (request.isSent()) {
```

Comment:
Writting this PR, I asked myself why `isSent` is called which make me feel that I maybe missed something :thinking:

Reply:
```
public void onSent() {
	for (MessageInterceptor interceptor : interceptors) {
		if (interceptor instanceof MessageInterceptor2) {
			MessageInterceptor2 postInterceptor = (MessageInterceptor2) interceptor;
			if (message instanceof Request) {
				postInterceptor.requestSent((Request) message);
			} else if (message instanceof Response) {
				postInterceptor.receiveResponse((Response) message);
			} else if (message instanceof EmptyMessage) {
				postInterceptor.emptyMessageSent((EmptyMessage) message);
			}
		}
	}
	message.setSent(true);
}
```
and

```
public void requestSent(Request request) {
	if (request.isSent()) {
		resentRequests.increment();
	} else {
		sentRequests.increment();
	}
}
```

The idea is simply: 
On transmission, the handler is called before "setSent(true)", so `sent` is `false`. 
On retransmission, `sent` is already `true` (set by the first transmission). 
Therefore `sent` could be used to distinguish between "retransmission" and "transmission". Alternatively, the `Message.duplicate` would be possible to use, if it gets also set on sending, currently it's only used for receiving.